src/app/_components/PageHeader.tsx

```diff
@@ -12,7 +14,7 @@ type CTAProps = {
   icon?: React.ReactNode
 }
 
-type ImageProps = {
+export type ImageProps = {
```

Comment:
I think we can centralize some of these under `_types` - Maybe under a file called `SharedProps` - the `cta` type could also go here, for example.